streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -250,6 +252,7 @@ public void shouldPutFetchRangeFromCache() {
         assertEquals(2, cache.size());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
This method also seems to be testing `get`, rather than `put(k,v)`, so maybe we can just migrate away from the deprecated method.

Reply:
removed